img/Project 3.html

```diff
@@ -0,0 +1,196 @@
+
+        <label for="lname">Last Name</label><br>
+        <input type="text" id="lname" name="lname" required=""><br>
+
+        <label for="birthday">Birthday</label><br>
+        <input type="date" id="birthday" name="birthday" required=""><br>
+
+        <label for="email">Email</label><br>
+        <input type="email" id="email" name="email" required=""><br>
+
+        <label for="pass">Password</label><br>
+        <input type="password" id="pass" name="pass" required="" minlength="6" maxlength="12"><br>
+
+      </fieldset>
+
+
+      <!-- Signup form (/ Interest selection) -->
+      <fieldset>
+        <legend>About your interest</legend>
+        <div class="themes-options">
+          <div class="option">
+            <input type="radio" id="3d-animation" value="3d-animation">
+            <label for="3d-animation">3d-animation</label>
+          </div>
+          <div class="option">
+            <input type="radio" id="branding" value="branding">
+            <label for="3branding">Branding</label>
+          </div>
+          <div class="option">
+            <input type="radio" id="illustration" value="illustration">
+            <label for="illustration">Illustration</label>
+          </div>
+          <div class="option">
+            <input type="radio" id="mobile-art" value="mobile-art">
+            <label for="mobile-art">Mobile Art</label>
+          </div>
+          <div class="option">
+            <input type="radio" id="print" value="print">
+            <label for="print">Print</label>
+          </div>
+          <div class="option">
+            <input type="radio" id="typography" value="typography">
+            <label for="typography">Typography</label>
+          </div>
+          <div class="option">
+            <input type="radio" id="web-design" value="web-design">
+            <label for="web-design">Web Design</label>
+          </div>
+          <div class="option">
+            <input type="radio" id="craft" value="craft">
+            <label for="craft">Craft</label>
+          </div>
+          <div class="option">
+            <input type="radio" id="photography" value="photography">
+            <label for="photography">Photography</label>
+          </div>
+          <div class="option">
+            <input type="radio" id="video" value="video">
+            <label for="video">video</label>
+          </div>
+        </div>
+      </fieldset>
+      <input class="submit-buttom" type="submit" value="submit">
+    </form>
+  </section>
+
+  <section class="accordion">
+    <!-- Accordion -->
+    <h1>FAQ</h1>
+    <div class="question1" id="section1">Section 1</div>
+    <div class="answer" id="answer">
+      <p>Lorem ipsum...</p>
+    </div>
+
+    <div class="question" id="section2">Section 2</div>
+    <div class="answer">
+      <p>Lorem ipsum...</p>
+    </div>
+  </section>
+  <script src="./Project 3_files/script.js.descarga"></script>
```

Comment:
script file should have extension `.js`, without anything else.
There is also another thing - I've noticed you use spanish words from time to time (in class name, or over here). We should avoid that - english language is like a latin of developers, we communicate between countries thanks to that. This rule should be applied to everything we write :)